pkg/config/config.go

```diff
@@ -39,6 +39,8 @@ type Config struct {
 	HostsDir         []string `toml:"hosts_dir"`
 	Experimental     bool     `toml:"experimental"`
 	HostGatewayIP    string   `toml:"host_gateway_ip"`
+
+	ExplicitInsecureRegistry bool `toml:"-"`
```

Comment:
Not sure about this. The struct is dedicated for TOML properties.

Reply:
Yeah, I was thinking about that.

We could use a different struct instead (or just an extra param) - but then we would have to modify the signature of a lot of functions in the path of `pull`, `push`, `login`.

Let me know what you prefer.

Reply:
Ok. Alternative proposal (overloading globalconfig) with the latest push.
LMK your thoughts.